examples/Interceptor/Client/ClientLoggerInterceptor.cs

```diff
@@ -132,5 +165,16 @@ private async Task<TResponse> HandleResponse<TResponse>(Task<TResponse> t)
             headers.Add("caller-machine", Environment.MachineName);
             headers.Add("caller-os", Environment.OSVersion.ToString());
         }
+
+        private static void LogError(Exception ex)
+        {
+            // Log error to the console.
+            // Note: Configuring .NET Core logging is the recommended way to log errors
+            // https://docs.microsoft.com/aspnet/core/grpc/diagnostics#grpc-client-logging
+            var initialColor = Console.ForegroundColor;
+            Console.ForegroundColor = ConsoleColor.Red;
+            Console.WriteLine($"Call error: {ex.Message}");
```

Comment:
```suggestion
            Console.WriteError($"Call error: {ex.Message}");
```

Or is there no particular need to specifically map to stderr here?